decidim-meetings/app/views/decidim/meetings/polls/questions/_closed_question.html.erb

```diff
@@ -1,7 +1,13 @@
-<summary><%= t(".question_results") %></summary>
+<summary>
+  <span><%= t(".question") %></span>
+  <span><%= t(".question_results") %></span>
+</summary>
 
 <div>
-  <p><%= translated_attribute(question.body) %></p>
+  <p class="h3"><%= translated_attribute(question.body) %></p>
```

Comment:
Why not an h3?

Reply:
To avoid accessibility issues related with the headings sequence. The page has no h2

Reply:
ok!